extern/iso_c_fortran_bmi/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 cmake_minimum_required(VERSION 3.12)
+project(iso_c_fortran_bmi VERSION 1.0.0 DESCRIPTION "Shared library for ISO_C_BINDING Fortran Libraries implementing BMI")
 enable_language( Fortran )
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../../cmake/")
 # Uncomment this and rebuild artifacts to enable debugging
-set(CMAKE_BUILD_TYPE Debug)
+#set(CMAKE_BUILD_TYPE Debug)
```

Comment:
The debug flags should be appropriately added only to debug build types, with something like the following generator expression.  This addes `-Wall` to all builds, but only the extra debug flags to the Debug build.
```cmake
add_compile_options(
  "-Wall"
  "$<$<CONFIG:DEBUG>:-g;-fbacktrace;-fbounds-check>"
)
```

Reply:
fixed with 2391f54